### PR TITLE
TESTING: do not merge test

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: staging/prometheus-operator/templates/prometheus/ingress.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+{{- if .Valjues.protheus.enabled }}
```
Comment (samvantran) [inline staging/prometheus-operator/templates/prometheus/ingress.yaml:1]: Protheus?

Comment (alejandroEsc) [reply staging/prometheus-operator/templates/prometheus/ingress.yaml:1]: on purpose

**Comment by dkoshkin** [comment]
Looks like the linter is still not failing as expected.

**Comment by alejandroEsc** [comment]
closing as this was used for testing.

**Pull request closed without merging**